controllers/Tuit2TagController.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file Controller RESTful Web service API for the tuit2tag resource
+ */
+
+import {Request, Response, Express} from "express";
+import Tuit2TagDao from "../daos/Tuit2TagDao";
+import Tuit2TagControllerI from "../interfaces/Tuit2TagControllerI";
+import Tuit2Tag from "../models/tags/Tuit2Tag";
+
+/**
+ * @class Tuit2TagController Implements RESTful Web service API for the
+ * Tuit2Tag resource.Defines the following HTTP endpoints:
+ * <ul>
+ *     <li>POST /api/tuit2tags to record that a new tuit2tag relationship has
+ *     been added to the database
+ *     </li>
+ *     <li>DELETE /api/tuit2tags/:tagID to record that a tuit2tag relationship
+ *     has been deleted
+ *     </li>
+ * </ul>
+ * @property {Tuit2TagDao} tuit2tagDao Singleton DAO implementing tags CRUD operations
+ * @property {Tuit2TagController} Tuit2TagController Singleton controller implementing
+ * RESTful Web service API
+ */
+
+export default class Tuit2TagController implements Tuit2TagControllerI {
+    private static tuit2tagDao: Tuit2TagDao = Tuit2TagDao.getInstance();
+    private static tuit2tagController: Tuit2TagController | null = null;
+    /**
+     * Creates a singleton controller instance
+     * @param {Express} app express instance to declare the RESTful API
+     * @return Tuit2TagController
+     */
+    public static getInstance = (app: Express): Tuit2TagController => {
+        if (Tuit2TagController.tuit2tagController === null) {
+            Tuit2TagController.tuit2tagController = new Tuit2TagController();
+            app.post("api/tuit2tags", Tuit2TagController.tuit2tagController.createTuit2Tag);
+            app.delete("api/tuit2tags/:t2tID", Tuit2TagController.tuit2tagController.deleteTuit2Tag);
+        }
+        return Tuit2TagController.tuit2tagController;
+    }
+
+    private constructor() {}
+
+/**
+ * Creates a new tuit2tag document in the database.
+ * @param {Request} req Represents request from client, including body
+ * containing the JSON object for the new tuit2tag to be inserted into
+ * the database
+ * @param {Response} res Represents response to client, including the
+ * body formatted as JSON containing the new tuit2tag that was inserted
+ * into the database
+ */
+createTuit2Tag = (req: Request, res: Response) =>
```

Comment:
This file doesn't work for me in postman. Suggested edits to createTuit2Tag:

    createTuit2Tag = (req: Request, res: Response) =>
        Tuit2TagController.tuit2tagDao.createTuit2Tag(req.params.tuitID, req.params.tagID)
            .then(tuit2tag => res.json(tuit2tag));

Reply:
or really it would just be line 55:

        Tuit2TagController.tuit2tagDao.createTuit2Tag(req.params.tuitID, req.params.tagID)